drivers/hmis/app/graphql/mutations/create_service_category.rb

```diff
@@ -1,32 +0,0 @@
-#  Copyright 2016 - 2024 Green River Data Analysis, LLC
```

Comment:
Ok, it still takes me minutes not seconds to reason through backwards compatibility, but I think this deletion (and deletion of the other mutations for `delete_` and `rename_`) is fine, since it's part of the frontend schema but not used. So, this version of the backend should be compatible with both the old and new version of the frontend.

Reply:
That's right!